### PR TITLE
dev: es6 + visual mods

### PR DESCRIPTION
**Pull request by aruznieto**
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

Issue Number: #73

## What is the new behavior?

ES6

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

I update the margin of the page to

### CODE REVIEW

File: backend/routes/network.js [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+import { defaultRules } from "../utils/constants.js";
+import { getZTAddress } from "../utils/zt-address.js";
 
 let ZT_ADDRESS = null;
 getZTAddress().then(function (address) {
   ZT_ADDRESS = address;
 });
 
 // get all networks
-router.get("/", auth.isAuthorized, async function (req, res) {
```
Comment (dec0dOS) [inline backend/routes/network.js]: I would prefer to import the whole module and refer to `isAuthorized()` as `auth.isAuthorized`. Thanks!
So as in other places

File: backend/routes/member.js [hunk 2]
```diff
@@ -36,33 +40,33 @@ router.get("/:mid", auth.isAuthorized, async function (req, res) {
...
 });
 
 // delete member
-router.delete("/:mid", auth.isAuthorized, async function (req, res) {
+router.delete("/:mid", isAuthorized, async function (req, res) {
   // @ts-ignore
   const nwid = req.params.nwid;
   const mid = req.params.mid;
-  member.deleteMemberAdditionalData(nwid, mid);
+  deleteMemberAdditionalData(nwid, mid);
```
Comment (dec0dOS) [inline backend/routes/member.js]: I would prefer to import the whole module and refer to `deleteMemberAdditionalData()` as `member.deleteMemberAdditionalData`. Thanks!
So as in other places

**Comment by dec0dOS** [comment]
Thanks a lot for your PR, @aruznieto!
Please check my code review.

**Comment by aruznieto** [comment]
> Thanks a lot for your PR, @aruznieto! Please check my code review.

Where is it?

**Review by dec0dOS — CHANGES_REQUESTED** [review]

**Comment by dec0dOS** [comment]
Apologies for the oversight, it appears I forgot to hit the "Finish" button on the code review page. Unfinished code reviews aren't visible to others.

**Comment by aruznieto** [comment]
> Apologies for the oversight, it appears I forgot to hit the "Finish" button on the code review page. Unfinished code reviews aren't visible to others.

Pushed, can you add dev: commit?

**Review by dec0dOS — CHANGES_REQUESTED** [review]
Seems like you have added the `package-lock.json` by mistake. Please delete it from the root. You should also remove croner from the root `package.json` file

**Comment by aruznieto** [comment]
> Seems like you have added the `package-lock.json` by mistake. Please delete it from the root. You should also remove croner from the root `package.json` file

Done. Sorry

**Comment by dec0dOS** [comment]
Everything else seems fine! Looks good to me. Have you had a chance to test the croner package to make sure it behaves exactly as expected?

Keep in mind that `node-cron` and `croner` are implemented quite differently under the hood

**Comment by dec0dOS** [comment]
Hmm, it looks like the developer is trying to provide an easy drop-in replacement alternative for node-cron: https://croner.56k.guru/migration/#switching-from-node-cron

Everything should be fine

**Comment by aruznieto** [comment]
> Hmm, it looks like the developer is trying to provide an easy drop-in replacement alternative for node-cron

Yes

**Comment by dec0dOS** [comment]
Thank you for collaborating, @aruznieto!

**Pull request merged**